### PR TITLE
Tests and small mods

### PR DESCRIPTION
**Pull request by JonathanFrey0000**

### CODE REVIEW

File: src/main/java/com/crumbs/fss/service/RestaurantService.java [hunk 1]
```diff
@@ -115,19 +94,26 @@ public Restaurant deleteRestaurant(Long id){
...
+        return temp;
+
+    }
     public Restaurant updateRestaurant(Long id, updateRestaurantDTO updateRestaurantDTO){
 
-        if(restaurantRepository.findById(id).isEmpty())
-            throw new EntityNotFoundException();
+        Restaurant temp = restaurantRepository.findById(id).orElseThrow(() -> new EntityNotFoundException());
 
         if(userDetailRepository.findUserByEmail(updateRestaurantDTO.getEmail())!=null)
```
Comment (elijahbrooks) [inline src/main/java/com/crumbs/fss/service/RestaurantService.java:111]: You could have your JPQL return an optional and if the entity isn't present throw an EntityNotFoundException

File: src/main/java/com/crumbs/fss/controller/MainController.java [hunk 2]
```diff
@@ -120,4 +123,7 @@ public Restaurant updateRestaurant(@PathVariable Long id, @Valid @RequestBody up
     public Restaurant deleteRestaurant(@PathVariable Long id){
         return restaurantService.deleteRestaurant(id);
     }
+
```
Comment (jim-brower) [inline src/main/java/com/crumbs/fss/controller/MainController.java:126]: change mapping to:
owners/{id}/restaurants

File: src/main/java/com/crumbs/fss/service/RestaurantService.java [hunk 3]
```diff
@@ -115,19 +94,26 @@ public Restaurant deleteRestaurant(Long id){
 
         return temp;
     }
+    public Restaurant deleteRestaurantOwnerRestaurant(Long id){
```
Comment (jim-brower) [inline src/main/java/com/crumbs/fss/service/RestaurantService.java:97]: In the future should support restaurant owner having multiple restaurants

File: src/test/java/com/crumbs/fss/controller/MainControllerTest.java [hunk 4]
```diff
@@ -57,6 +67,21 @@ void getMenuItems() throws Exception {
 //                .contentType(MediaType.APPLICATION_JSON))
 //                .andExpect(status().isOk());
     }
+    @Test
+    void getRestaurantss() throws Exception {
+
+        mockMvc.perform(get("/restaurantss")
```
Comment (jim-brower) [inline src/test/java/com/crumbs/fss/controller/MainControllerTest.java:73]: funny test

**Review by jim-brower — CHANGES_REQUESTED** [review]
looks good, just change mapping on endpoint mentioned.

**Pull request merged**